addons/web/static/tests/fields/relational_fields_tests.js

```diff
@@ -4314,6 +4314,97 @@ QUnit.module('relational_fields', {
         form.destroy();
     });
 
+    QUnit.test('onchange many2many in one2many list editable', function (assert) {
```

Comment:
It goes through the code but looked that it had not changed, I have extended for the particular behavior of the focus.